docs/docusaurus.config.js

```diff
@@ -16,9 +16,16 @@ module.exports = {
   organizationName: 'facebookexperimental', // Usually your GitHub org/user name.
   projectName: 'Recoil', // Usually your repo name.
   themeConfig: {
-     algolia: {
+    announcementBar: {
+      id: 'black_lives_matter',
+      content:
+        '<div class="blm-bar">Black Lives Matter. <a style="color: rgb(97, 218, 251)" target="_blank" href="https://support.eji.org/give/153413/#!/donation/checkout">Support the Equal Justice Innitiative</a></div>',
```

Comment:
Should be "initiative"

Reply:
Oh man I copy-pasted this I have no idea how that happened 😅  good catch!